### ISSUE TITLE
NTR: Skin biopsy: subcutaneous arterioles thrombosis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9386]
New term request Skin biopsy: subcutaneous arterioles thrombosis (elthat_150807073134):   UMLS: 0262992

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000075
- label: Subcutaneous arteriolar thrombosis
- parents: [Abnormal thrombosis (HP:0001977)](https://hpo.jax.org/app/browse/term/HP:0001977)
- pmid: 
- def: Fibrin thrombi inside subcutaneous arterioles.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9386
- synonyms:
